### PR TITLE
add description of all samples to main readme

### PR DESCRIPTION
**Pull request by snedamle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,52 @@
 
 This repository contains multiple conclave sample applications which is intended to help
 developers to get started with the conclave platform and understand different features of
```
Comment (danibentrup) [inline README.md]:
```suggestion
This repository contains multiple Conclave sample applications which is intended to help
developers to get started with Conclave and understand different features of
```

File: README.md [hunk 2]
```diff
@@ -7,10 +7,52 @@
...
-the platform. To learn more about the Conclave platform please visit our official 
+the platform. To learn more about the Conclave platform please visit our official
 [documentation](https://docs.conclave.net/).
 
+### [EventManager](./EventManager):
+Event Manager implements the idea of an enclave that can simultaneously host a collection of 'Computations'. These computations are initiated by a Conclave client, have a name and a type, have a defined set of 'participants' and a 'quorum' that must be reached before a result (or results) can be obtained. Supported types are average (the average of the submissions is returned), maximum and minimum (the identity of the submitter of the highest or lowest value is returned) and "key matcher" (described below)
```
Comment (danibentrup) [inline README.md]:
```suggestion
Event Manager implements the idea of an enclave that can host a collection of simultaneous 'computations'. These computations are initiated by a Conclave client, have a name and a type, have a defined set of 'participants', and a 'quorum' that must be reached before results can be obtained. Supported types are average (the average of the submissions is returned), maximum and minimum (the identity of the submitter of the highest or lowest value is returned), and 'key matcher' (described below).
```

File: README.md [hunk 3]
```diff
@@ -7,10 +7,52 @@
...
+The idea is that an enclave of this type could be used to host simple one-off multi-party computations (eg five employees wanting to know the average of their salaries, or ten firms wanting to know who had the lowest sales last month without revealing what that figure was).
```
Comment (danibentrup) [inline README.md]:
```suggestion
The idea is that an enclave of this type could be used to host simple one-off multi-party computations (e.g., five employees wanting to know the average of their salaries, or ten firms wanting to know who had the lowest sales last month without revealing what that figure was).
```

File: README.md [hunk 4]
```diff
@@ -7,10 +7,52 @@
...
+
+### [Column Profiling](./column-profiling):
+This sample does Column Profiling on the dataset and
+returns the frequency distribution as an output. Column profiling is one of the methods used in data profiling.
+This [link](https://www.alooma.com/blog/what-is-data-profiling) explains what data profiling and column profiling means
```
Comment (danibentrup) [inline README.md]:
```suggestion
This [article](https://www.alooma.com/blog/what-is-data-profiling) explains what data profiling and column profiling means
```

File: README.md [hunk 5]
```diff
@@ -7,10 +7,52 @@
...
+and explains its uses and applications.
+
+### [Conclave Auction](./conclave-auction):
+This sample allows bidders to confidentially submit their bids to an enclave (A protected region of memory which cannot be accessed by the OS, Kernel or BIOS). The bids are processed confidentially in the enclave and the result is returned to concerned parties.
+
+### [Conclave Corda Trade](./conclave-corda-trade):
+This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes servers as clients which send encryped order from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
```
Comment (danibentrup) [inline README.md]:
```suggestion
This application serves as a demo for building a confidential trading system based on Corda and Conclave. An exchange Corda node would serve as a host which runs the enclave, while broker nodes serve as clients which send encrypted orders from their end-clients which are matched in the enclave and trades generated are recorded in all relevant participants ledgers.
```

File: README.md [hunk 6]
```diff
@@ -7,10 +7,52 @@
...
+
+### [Psi](./psi-sample):
+PSI problem refers to the problem of determining the common elements from the intersection of two sets without leaking or disclosing any additional information of the remaining elements of the either sets.
```
Comment (danibentrup) [inline README.md]:
```suggestion
PSI problem refers to the problem of determining the common elements from the intersection of two sets without leaking or disclosing any additional information of the remaining elements of either sets.
```

File: README.md [hunk 7]
```diff
@@ -7,10 +7,52 @@
...
+Using this sample will see how Conclave (based on Intel SGX) can be a new tool to solve the private set intersection (PSI) problem.
```
Comment (danibentrup) [inline README.md]:
```suggestion
Using this sample will demonstrate how Conclave (based on Intel SGX) can be a new tool to solve the private set intersection (PSI) problem.
```

File: README.md [hunk 8]
```diff
@@ -7,10 +7,52 @@
...
+
+### [Tribuo Classification](./tribuo-classification):
+Using this sample will see how Conclave (based on Intel SGX) can be used in training a ML model. We will use tribuo Java ML library to load the AI model.
```
Comment (danibentrup) [inline README.md]:
```suggestion
Using this sample will show how Conclave (based on Intel SGX) can be used in training a ML model. We will use Tribuo Java ML library to load the AI model.
```

File: README.md [hunk 9]
```diff
@@ -7,10 +7,52 @@
...
+Most of the hospitals/doctors have lot of their patients' data, which can be used to determine whether a tumor is malignant or begnin.
```
Comment (danibentrup) [inline README.md]:
```suggestion
For example, hospitals have patients' data which can be used to determine whether a tumour is malignant or benign.
```

File: README.md [hunk 10]
```diff
@@ -7,10 +7,52 @@
...
+Such data can be used to train an AI model. Once a model is trained, this model can be used to predict if a given tumor is malignant
+or begnin given certain input attributes.
+
+### [Tribuo Tutorials](./tribuo-tutorials):
+Tribuo is a Java machine learning library, which makes it well suited to run with Conclave. This sample provides tools for classification, regression, clustering, model development, and more.
+This sample shows you how you can use the [Tribuo Java ML](https://tribuo.org/learn/4.0/tutorials/) library to load, train many models like classification models, regression models, clustering models etc.
```
Comment (danibentrup) [inline README.md]:
```suggestion
This sample shows you how you can use the [Tribuo Java ML](https://tribuo.org/learn/4.0/tutorials/) library to load and train models like classification models, regression models, clustering models etc.
```

File: README.md [hunk 11]
```diff
@@ -7,10 +7,52 @@
...
+
+### [Database Enclave](./psi-sample):
+With the new 3rd Gen Intel Xeon Scalable Processors supporting 1 TB of enclave memory, 
+setting up database inside and enclave is very well possible. This sample shows how persistence is used to create a database inside an enclave and save data into it, how to create a
```
Comment (danibentrup) [inline README.md]:
```suggestion
setting up a database inside an enclave is very well possible. This sample shows how persistence is used to create a database inside an enclave and save data into it as well as how to create a
```

File: README.md [hunk 12]
```diff
@@ -7,10 +7,52 @@
...
+table/insert records into it/select records from the table. This also shows how persisted records can be retrieved by
+the enclave once the host is re-started.
+
+### Initial SetUp
+
+In order to compile, and run these samples successfully, kindly download the conclave SDK and update
```
Comment (danibentrup) [inline README.md]:
```suggestion
In order to compile, and run these samples successfully, kindly download the [conclave SDK](https://www.conclave.net/get-conclave/) and update 
```

File: README.md [hunk 13]
```diff
@@ -7,10 +7,52 @@
...
+or begnin given certain input attributes.
+
+### [Tribuo Tutorials](./tribuo-tutorials):
+Tribuo is a Java machine learning library, which makes it well suited to run with Conclave. This sample provides tools for classification, regression, clustering, model development, and more.
+This sample shows you how you can use the [Tribuo Java ML](https://tribuo.org/learn/4.0/tutorials/) library to load, train many models like classification models, regression models, clustering models etc.
+
+### [Database Enclave](./psi-sample):
+With the new 3rd Gen Intel Xeon Scalable Processors supporting 1 TB of enclave memory, 
+setting up database inside and enclave is very well possible. This sample shows how persistence is used to create a database inside an enclave and save data into it, how to create a
+table/insert records into it/select records from the table. This also shows how persisted records can be retrieved by
```
Comment (danibentrup) [inline README.md]:
```suggestion
table, insert records into it, and select records from the table. This also shows how persisted records can be retrieved by
```

**Comment by snedamle** [comment]
Review comments addressed. Merging the same.

**Pull request merged**